man/systemd.network.xml

```diff
@@ -902,6 +902,17 @@
           </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Protocol=</varname></term>
+          <listitem>
+            <para>The Protocol identifier for the route. Takes a number between 0 and 255 or the special values
+            <literal>kernel</literal>, <literal>boot</literal> and <literal>static</literal>. Values of protocol
+            greater than equal to <varname>static</varname> are not interpreted by kernel. They are just passed
+            from user and back as is. It will be used by hypothetical multiple routing daemons. Defaults to
+            <literal>static</literal>.
```

Comment:
Hm, it seems that the relevant information is here, but somewhere around the third sentence I stop understanding the text (e.g. "greater than equal to" was probably meant to be "greater than or equal to", but even with that change, I don't see how "static" can be larger or smaller than something else).

@zktech you requested this feature, so you probably know it does ;) Can you maybe help with the text?